s3/sc-portfolio-s3-basic.yaml

```diff
@@ -66,76 +22,34 @@ Parameters:
     Type: String
     Description: Last updated date and time of portfolio
     Default: ''
-Conditions:
-  CreateLaunchConstraint: !Equals
-    - !Ref 'LaunchRoleName'
-    - ''
-  CondPrincipalRoleName1: !Not
-    - !Equals
-      - !Ref 'PrincipalRoleName1'
-      - ''
-  CondPrincipalGroupName1: !Not
-    - !Equals
-      - !Ref 'PrincipalGroupName1'
-      - ''
-  CondPrincipalRoleName2: !Not
-    - !Equals
-      - !Ref 'PrincipalRoleName2'
-      - ''
-  CondPrincipalGroupName2: !Not
-    - !Equals
-      - !Ref 'PrincipalGroupName2'
-      - ''
 Resources:
   SCS3portfolio:
     Type: AWS::ServiceCatalog::Portfolio
     Properties:
       ProviderName: !Ref 'PortfolioProvider'
       Description: !Ref 'PorfolioDescription'
       DisplayName: !Ref 'PorfolioName'
-  LaunchConstraintRole:
-    Type: AWS::CloudFormation::Stack
-    Condition: CreateLaunchConstraint
-    Properties:
-      TemplateURL: !Sub '${RepoRootURL}iam/sc-s3-launchrole.yaml'
-      TimeoutInMinutes: 5
   LinkEndusersRole:
     Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
```

Comment:
How about leaving the condition statements the restrict which roles can execute products? We might use these conditions if we start differentiating roles